### DISCUSSION TITLE
Folders

### DISCUSSION THREAD

**Discussion by iu14472** [tanujnotes/Olauncher#431] [Ideas]
![1](https://github.com/tanujnotes/Olauncher/assets/133242157/440b20e5-686d-4c18-a46a-5471086c7e7e)

**Comment by xi7** [comment]
u can do it using rename, like this:
⭐telegram
⭐messenger
⭐Viber

but dev rename method is boring almost unusable.

**Comment by iu14472** [comment]
![Out1](https://github.com/tanujnotes/Olauncher/assets/133242157/3fbb525d-3e9d-4099-9b4a-35162cf28606)
![Out2](https://github.com/tanujnotes/Olauncher/assets/133242157/5638573f-3e09-44a2-b051-e592b06faedd)
